src/ecs/explorer/ecsServiceNode.ts

```diff
@@ -39,9 +39,14 @@ export class EcsServiceNode extends AWSTreeNodeBase implements AWSResourceNode {
         return await makeChildrenNodes({
             getChildNodes: async () => {
                 const containerNames = await this.ecs.getContainerNames(this.service.taskDefinition!)
-                return containerNames.map(
-                    name => new EcsContainerNode(name, this.name, this.parent.arn, this.ecs, this)
-                )
+                const childNodes = containerNames.map(name => {
+                    return new EcsContainerNode(name, this.name, this.parent.arn, this.ecs, this)
+                })
+                for (const node of childNodes) {
+                    await node.updateRunningTasks()
+                }
```

Comment:
I wonder if it's worth doing something like:
```
const childPromises = [] // unsure what the type is
for (const node of childNodes) {
    childPromises.push(node.updateRunningTasks())
}
await Promises.all(childPromises)
```

You'd load faster (since all of the promises will run in parallel instead of in series) but then you'll have to worry about throttling if there are large amounts of child nodes. Might be worth keeping an eye on...maybe add telemetry to the length of this call (get amt of child nodes and length of `getChildNodes` call?)

Reply:
Our S3 buckets are retrieved with `Promise.all` as well. Seems to handle ~100 buckets fine, though I'd wager there would be small benefits if we chunked into groups of 5-10. The SDKs will retry throttling errors (up to a point) without us knowing so it can be difficult to determine the impact.